data/human/transport missions.txt

```diff
@@ -3582,3 +3582,297 @@ mission "Hauler VI cargo prototype 3"
 			`The Testament enters the planet's atmosphere at high speed and lands somewhat uncontrolled near a waste processing plant. The cargo hold is glowing red hot and some areas in the cargo hold appear to be on fire.`
 			`	Veronica disembarks, walks over to you, and apologizes for the difficult trip. She explains that the cooling systems failed, causing the sushi calamity you experienced during the multiple legs of your journey.`
 			`	You are paid <payment> and she asks you not to talk about this escort job to anybody. You leave the docks quickly in order to escape the horrible smell of overheated fish that is coming from Veronica and her ship.`
+
+
+
+mission "James Medicine [0]"
+	minor
+	name "Medicine from <planet>"
+	description "James's doctors have asked you to bring some newly-developed remedy from <planet> to <origin> as quickly as possible, even though they are not really sure what his illness is."
+	deadline
+	source "Hestia"
+	destination "Nimbus"
+	to offer
+		has "main plot completed"
+		or
+			has "Intro [3 Transport]: done"
+			has "Intro [3 Freighter]: done"
+			has "Intro [3 Interceptor]: done"
+	on offer
+		conversation
+			`Finding yourself on <origin>, you smile at the memory of having dropped James here after he gave you some basic spaceflight instructions. Do you wish to visit him and see how he has been faring?`
+			choice
+				`	(Yes.)`
+				`	(Not right now.)`
+					defer
+			`	Unlike some other planets, where it is practically impossible to find a person without having contact information, <origin> has a planetary database. You eagerly walk over to the nearest information desk and wait for the operator to make a few calls. At last, you are directed to a small, respected establishment, the Pilot Veteran Clinic. You can only wonder what James is doing there.`
+			`	The clinic is housed in a small, modern-looking building, which looks cunningly similar to a giant Firebird. Inside, the rooms are arranged to resemble bunk rooms on a spaceship, likely so that the patients would feel right at home.`
```

Comment:
Not sure if `cunningly` is fitting here.

Reply:
As per https://www.thefreedictionary.com/cunning:
> 2. Executed with or exhibiting ingenuity.

The clinic is not just an ordinary square/rectangular building. It was built from the ground in order to be a clinic for veteran pilots. What better way to show that off, apart from making it shaped like a iconic ship which every veteran pilot would have flown?

I'd be willing to accept a better adverb there. But I still think shows some ingenuity to architecture a building after its purpose.

Reply:
"Cunningly" has a negative connotation to it, which makes it off-putting here. Maybe the second half of the phrase could be rewritten to "designed to look like a giant Firebird"? That conveys the intent to make it look like a Firebird while removing the "cunningly".